src/models/sourcecodemodel.cpp

```diff
@@ -51,6 +51,14 @@ void SourceCodeModel::setDisassembly(const DisassemblyOutput& disassemblyOutput,
     if (!file.open(QIODevice::ReadOnly | QIODevice::Text))
         return;
 
+    QString sourceCode = QString::fromUtf8(file.readAll());
```

Comment:
commenting here b/c github doesn't allow me to comment on the commit message: you don't call reverse/rotate anymore. just say that it makes searching easier and be done with it